### PR TITLE
Stop reusing `send` in `raise`

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/core/test/actions.test.ts [hunk 1]
```diff
@@ -2734,6 +2733,67 @@ describe('raise', () => {
       expect(service.getSnapshot().value).toEqual('a');
     }, 50);
   });
+
+  it('should accept event expression', () => {
```
Comment (Andarist) [inline packages/core/test/actions.test.ts:2737]: TODO: add more tests

File: packages/core/test/types.test.ts [hunk 2]
```diff
@@ -253,23 +253,23 @@ describe('Raise events', () => {
               {
                 actions: raise({
                   type: 'ALOHA'
-                }),
+                } as any),
```
Comment (Andarist) [inline packages/core/test/types.test.ts:256]: improvements to `raise` typings will be in a separate PR

**Review by davidkpiano — APPROVED** [review]
Code LGTM
